### PR TITLE
Fix done button color in the dark mode

### PR DESCRIPTION
**Pull request by SaeedNoshadi89**
This is related to #46 
fix the ` Done`  button text color in the dark mode state.

### CODE REVIEW

**Comment by ricknout** [comment]
The fix should rather be to replace `Button` with `NiaFilledButton`[https://github.com/android/nowinandroid/blob/main/core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/component/Button.kt#L54]. `Button(colors = ...)` and `Text(color = ...)` also needs to be removed. `NiaFilledButton` has the correct colors for disabled state.

**Comment by SaeedNoshadi89** [comment]
> The fix should rather be to replace `Button` with `NiaFilledButton`[https://github.com/android/nowinandroid/blob/main/core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/component/Button.kt#L54]. `Button(colors = ...)` and `Text(color = ...)` also needs to be removed. `NiaFilledButton` has the correct colors for disabled state.

I changed to NiaFilledButton

File: feature-foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt [hunk 1]
```diff
@@ -275,7 +276,7 @@ private fun LazyListScope.InterestsSelection(
                     horizontalArrangement = Arrangement.Center,
                     modifier = Modifier.fillMaxWidth()
                 ) {
-                    Button(
+                    NiaFilledButton(
```
Comment (ricknout) [inline feature-foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt:279]: The `colors` override still needs to be removed from here

**Comment by ricknout** [comment]
Thanks! Can you please confirm it looks okay now with a screenshot in dark mode / button disabled?

**Comment by SaeedNoshadi89** [comment]
> Thanks! Can you please confirm it looks okay now with a screenshot in dark mode / button disabled?

Sure, I will take screenshots in light mode and dark mode.

<table>
<td align="center"> </td>
<td align="center"> </td>
</table>

<table>
<td align="center"> </td>
<td align="center"> </td>
</table>

**Review by ricknout — APPROVED** [review]

**Review by ricknout — APPROVED** [review]

**Comment by SaeedNoshadi89** [comment]
> Please run `spotlessApply` on your code to fix formatting errors!

@JolandaVerhoef Done

**Pull request merged**